rgbd_head_pitch_controller:
  controller:
    module: joint_position_controller
    package: dynamixel_controllers
    type: JointPositionController
  joint_name: rgbd_head_joint
  joint_speed: 0.3
  motor:
    id: 21
    init: 710 # tick para dejar calibracion ~correcta en 40-60 [deg] de inclinacion
    max: 950
    min: 660